lib/app/components/elements/checkboxes.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:f_kit_x/app/components/component_list.dart';
+import 'package:f_kit_x/components/checkboxes/checkbox_demo.dart';
+import 'package:flutter/material.dart';
+
+class Checkboxes extends StatefulWidget {
+  const Checkboxes({Key? key}) : super(key: key);
+
+  @override
+  State<Checkboxes> createState() => _CheckboxesState();
+}
+
+class _CheckboxesState extends State<Checkboxes> {
+  late bool isChecked = false;
```

Comment:
isChecked variable was initialized using late bool  which would throw a runtime error if it was not initialized before it was accessed.

Reply:
its initialized by default,  so no exception would thrown here. but since its getting initialized by definition, `late` is obsolete